src/views/Deposit/DepositDash.tsx

```diff
@@ -85,26 +85,19 @@ const DepositDashNative: React.FC<DepositDashNativeProps> = ({ token }) => {
   const { data: reserves } = useAllReserveTokensWithData();
   const { data: tokenBalance } = useUserAssetBalance(token);
   const { data: wnative } = useWrappedNativeDefinition();
-  const reserve = React.useMemo(
-    () =>
-      reserves?.find(
```

Comment:
So you scrapped the case-preserving perf optimization? That's fine as long as you're aware you did.